packages/core/helper-plugin/webpack.config.js

```diff
@@ -1,55 +1,43 @@
-const webpack = require('webpack');
-const { ESBuildMinifyPlugin } = require('esbuild-loader');
-const browserslistToEsbuild = require('browserslist-to-esbuild');
+const webpack = require("webpack");
+const { ESBuildMinifyPlugin } = require("esbuild-loader");
+const browserslistToEsbuild = require("browserslist-to-esbuild");
 
-const packageJson = require('./package.json');
+const packageJson = require("./package.json");
 
 const nodeModules = [];
-[
-  ...Object.keys(packageJson.dependencies || {}),
-  ...Object.keys(packageJson.peerDependencies || {}),
-  ...Object.keys(packageJson.devDependencies || {}),
-].forEach((module) => {
+Object.keys(packageJson.dependencies).forEach((module) => {
   nodeModules.push(new RegExp(`^${module}(/.+)?$`));
 });
 
-module.exports = {
+/** @type {Omit<import('webpack').Configuration, 'output'>} */
+const baseConfig = {
   entry: `${__dirname}/lib/src/index.js`,
   externals: nodeModules,
   mode: process.env.NODE_ENV,
-  devtool: process.env.NODE_ENV === 'production' ? false : 'eval-source-map',
+  devtool: process.env.NODE_ENV === "production" ? false : "eval-source-map",
   optimization: {
-    minimize: process.env.NODE_ENV === 'production',
+    minimize: process.env.NODE_ENV === "production",
     minimizer: [
       new ESBuildMinifyPlugin({
-        target: 'es2015',
+        target: "es2015",
       }),
```

Comment:
Should this use the relative browserslist target? We can also do this in a follow-up if you prefer.

Reply:
that should have been done in the previous PR – i missed the minifier 😓